modules/load_balancer/main.tf

```diff
@@ -1,6 +1,14 @@
+resource "random_id" "tg_name_suffix" {
```

Comment:
I see _what_ is going on but some comment for _why_ would be good.

Reply:
done